src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  } else if (days >= 3 && days < 6) {
+    return 100;
+  } else if (days === 6) {
+    return 220;
+  } else {
+    return 230;
```

Comment:
This condition should also be based on a calculation rather than a hardcoded value. Additionally, it does not account for the possibility of renting for more than 7 days, where the daily rate should still apply.